Playbook.yaml

```diff
@@ -0,0 +1,84 @@
+- name:  App - Java with Maven - Ubuntu Linux
+  hosts: servers
+  become: yes
+  remote_user: user
+  gather_facts: no
+  
+  vars:
+    maven_major_version: 3
+    maven_version: 3.8.6
+  
+  tasks:
+    - name: Install Default Java JRE/JDK Pkg
+      apt:
+        pkg:
+        - default-jre
+        - default-jdk
+
+    - name: Create Maven Directory
+      file:
+        path: /usr/local/apache-maven
+        state: directory
+        mode: '0775'
+        recurse: yes
+        owner: root
+        group: root
+
+    - name: Download and Unarchive Maven
+      unarchive:
+        src: https://dlcdn.apache.org/maven/maven-{{ maven_major_version }}/{{ maven_version }}/binaries/apache-maven-{{ maven_version }}-bin.tar.gz
+        dest: /usr/local/apache-maven
+        remote_src: yes
+
+    - name: Config ~/.bashrc
+      blockinfile:
+        create: yes
+        path: ~/.bashrc #/root/.bashrc
+        block: |
+          export M2_HOME=/usr/local/apache-maven/apache-maven-{{ maven_version }}
+          export M2=$M2_HOME/bin
+          export MAVEN_OPTS="-Xms256m -Xmx512m"
+          export PATH=$M2:$PATH
+
+    - name: Create Directory for Project
+      file:
+        path: /HelloWorldMaven
+        state: directory
+        mode: '0775'
+        recurse: yes
+        owner: root
+        group: root
+
+    - name: Clone App Repository
+      git:
+        repo: https://github.com/acmano7/java-hello-world.git
+        dest: /HelloWorldMaven
+        update: yes
+        force: yes
+
+    - name: Create Java App
+      shell: . ~/.bashrc && mvn install -f /HelloWorldMaven/pom.xml
+      
+    - name: Create Systemd Service for App
+      blockinfile:
+        create: yes
+        path: /etc/systemd/system/java-hello-world.service
+        block: |
+          [Unit]
+          Description=Java App
+
+          [Service]
+          ExecStart=/bin/bash -c "while true; do java -cp /HelloWorldMaven/target/jb-hello-world-maven-0.2.0.jar hello.HelloWorld > /tmp/java_app.txt && sleep 2; done"
+
```

Comment:
what about restarting and stopping the service?

Reply:
This is a script, so stop and start are ok for now.